app/uk/gov/hmrc/gform/controllers/FormTemplates.scala

```diff
@@ -103,7 +103,15 @@ class FormTemplates()(
       }
   }
 
-  def delete(formTypeId: FormTypeId, version: Version) = Action.async { implicit request =>
-    Future.successful(NotImplemented)
+  def delete(formTypeId: FormTypeId, version: String) = Action.async { implicit request =>
```

Comment:
The forms are stored in a separate collection, deleting the template means you can't access the form but the data is not deleted from the forms collection